Languages/french.lang

```diff
@@ -529,7 +529,7 @@ msgid "Could not write file."
 msgstr "Écriture du fichier impossible."
 
 msgid "Could not write to:"
-msgstr "Écriture impossible vers:"
+msgstr "Écriture impossible vers :"
 
 msgid "Cover Action"
 msgstr ""
```

Comment:
Action de jaquette? action sur les jaquettes?

I haven’t search for the context

Reply:
don't understand